test/supertest.factory.ts

```diff
@@ -1,8 +1,7 @@
-import { Application } from 'express'
```

Comment:
Can we delete this file? It's just a wrapper around supertest for no benefit atm.